invenio_app_ils/literature/covers_builder.py

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2018-2020 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Document configuration callbacks."""
+
+from flask import url_for
+
+
+def build_ils_demo_cover_urls(metadata):
+    """Build working ulrs for demo data."""
+    cover_metadata = metadata.get("cover_metadata", {})
+    isbn = cover_metadata.get("ISBN", "")
+    if isbn:
+        return build_openlibrary_urls(isbn)
+    return build_placeholder_urls()
+
+
+def build_openlibrary_urls(isbn):
+    """Build Open Library urls."""
+    url = "http://covers.openlibrary.org/b/isbn"
+    return {
+        "small": "{url}/{isbn}-S.jpg".format(url=url, isbn=isbn),
+        "medium": "{url}/{isbn}-M.jpg".format(url=url, isbn=isbn),
+        "large": "{url}/{isbn}-L.jpg".format(url=url, isbn=isbn),
+    }
+
+
+def build_placeholder_urls():
+    """Build urls for default cover placeholders."""
+    image_path = url_for(
+        "invenio_app_ils_mail.static",
+        filename="images/placeholder.png",
+        _external=True
+    )
+    return {
+        "small": image_path,
```

Comment:
I don't know if @kprzerwa wants to comment here, but we were chatting that maybe we want to know in the UI if we are displaying a real cover or a placeholder, to allow even more customization? A solution would be just to add an extra field `is_placeholder: true/false`...